### DISCUSSION TITLE
How fix Isort

### DISCUSSION THREAD

**Discussion by novak24611** [Kyrluckechuck/TFT-Bot#255] [General]
![Messenger_creation_bd2b0d2a-2b56-4559-96a3-baaadac2aa85](https://github.com/Kyrluckechuck/TFT-Bot/assets/64787005/42dc1527-03e9-4cc9-be3e-185cbf1d2a49)

WARNING: isort 5.13.2 does not provide the extra 'pipfile-deprecated-finder'

WARNING: isort 5.13.2 does not provide the extra 'requirements-deprecated-finder

**Comment by Kyrluckechuck** [comment]
Is this impacting ability to run the bot, or is it just a warning? I believe we're not actually using these features at present

**Reply by novak24611** [reply]
Since seeing this announcement, I've seen almost no more buying champions and automatically surrendering
